test/functional/fixtures/api/es-next/resize-window/testcafe-fixtures/resize-window-test.js

```diff
@@ -0,0 +1,56 @@
+// NOTE: to preserve callsites, add new tests AFTER the existing ones
+import { Hybrid } from 'testcafe';
+import { expect } from 'chai';
+
+
+fixture `Resize window`
+    .page `http://localhost:3000/api/es-next/resize-window/pages/index.html`;
+
+const getWindowWidth  = Hybrid(() => window.innerWidth);
+const getWindowHeight = Hybrid(() => window.innerHeight);
+const iPhoneSize      = { width: 480, height: 320 };
+
+test('Resize the window', async t => {
```

Comment:
maybe it's better to use `.beforeEach` and `.afterEach` for restoring browser size?
```js
var initialWindowWidth = null;
var initialWindowHeight = null;

...beforeEach(async t => {
        initialWindowWidth = await getWindowWidth();
        initialWindowHeight = await getWindowHeight();
    })
    .afterEach(async t => {
        await t.resizeWindow(initialWindowWidth , initialWindowHeight );
    });
```